src/lesson3/Task20.java

```diff
@@ -0,0 +1,24 @@
+package lesson3;
+
+import java.util.StringTokenizer;
+
+//Имеется строка с текстом. Вывести текст, составленный из последних букв
+//всех слов.
+public class Task20 {
```

Comment:
In general OK< but I would suggest better variable naming:
st -> tokenizer
str -> text
n -> character (or ch, at least)

Reply:
fixed it